src/main/java/com/fastmail/jstpierre/petmanager/consumers/Consumers.java

```diff
@@ -0,0 +1,20 @@
+package com.fastmail.jstpierre.petmanager.consumers;
+
+import com.fastmail.jstpierre.petmanager.entities.petowners.PetOwner;
+import com.fastmail.jstpierre.petmanager.entities.pets.Pet;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+public class Consumers {
+
+    public static final BiConsumer<PetOwner, String> newOwnerAddress = PetOwner::setStreetAddress;
+
+    public static final BiConsumer<HashMap<PetOwner, List<Pet>>, PetOwner> addPetsToMap
+            = (map, owner) -> {
+
+        List<Pet> pets = owner.getPets();
+        map.put(owner, pets);
```

Comment:
Peut être fait en une ligne :
```suggestion
        map.put(owner, owner.getPets());
```

Reply:
Ça me permet de retirer mes surrounding curly brackets, aussi.